src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -782,8 +782,11 @@
     <value>Windows default</value>
     <comment>Windows refers to the Operating system</comment>
   </data>
-  <data name="Windows_Color_Settings.Content" xml:space="preserve">
+  <data name="Windows_Color_Settings.Text" xml:space="preserve">
     <value>Windows color settings</value>
     <comment>Windows refers to the Operating system</comment>
   </data>
-</root>
+  <data name="KBM_KeysCannotBeRemapped.Text" xml:space="preserve">
+    <value>View what cannot be remapped</value>
```

Comment:
Lets add in a comment moving forward to strings to aid in localization.
<comment>This is a link that will discuss what is and is not possible for Keyboard manager to remap</comment>